### PR TITLE
Fixed Bug in Equation For Intersection of Line and Plane

### PR DESCRIPTION
**Pull request by viclai**
The equation to find the value of ’t' for the intersection of a line and a plane does not appear to be entirely correct.  Please double-check.

### CODE REVIEW

**Comment by josdejong** [comment]
Thanks for this fix @viclai!

Can you please add a unit test to prevent any future regressions?

**Comment by viclai** [comment]
Okay, I added the unit test.  @josdejong, let me know if you need me to do anything else related with this bug.

File: lib/function/geometry/intersect.js [hunk 1]
```diff
@@ -125,7 +125,7 @@ function factory (type, config, load, typed) {
   }
 
   function _intersectLinePlane(x1, y1, z1, x2, y2, z2, x, y, z, c){
-    var t = (c - x1*x - y1*y - z1*z)/(x2*x + y2*y + z2*z - x1 - y1 - z1);
+    var t = (c - x1*x - y1*y - z1*z)/(x2*x + y2*y + z2*z - x1*x - y1*y - z1*z);
```
Comment (viclai) [inline lib/function/geometry/intersect.js]: The correct derivation is as follows.

x * (x1 + t(x2 - x1)) + y *(y1 + t(y2 - y1)) + z * (z1 + t(z2 - z1)) = c
(x1 * x) + (x2 * x * t) - (x1 * x * t) + (y1 * y) + (y2 * y * t) - (y1 * y * t) + (z1 * z) + (z2 * z * t) - (z1 * z * t) = c
(x2 * x * t) - (x1 * x * t) + (y2 * y * t) - (y1 * y * t) + (z2 * z * t) - (z1 * z * t) = c - (x1 * x) - (y1 * y) - (z1 * z)
t * ((x2 * x) - (x1 * x) + (y2 * y) - (y1 * y) + (z2 * z) - (z1 * z)) = c - (x1 * x) - (y1 * y) - (z1 * z)
t = (c - (x1 * x) - (y1 * y) - (z1 * z)) / ((x2 * x) - (x1 * x) + (y2 * y) - (y1 * y) + (z2 * z) - (z1 * z))
t = (c - x1*x - y1 * y - z1*z) / (x2*x + y2*y + z2*z - x1*x - y1*y - z1*z)

**Comment by josdejong** [comment]
Thanks for adding tests! I will merge your PR now.

**Pull request merged**